@navikt/core/react/src/form/file-upload/FileUpload.tsx

```diff
@@ -0,0 +1,161 @@
+import { UploadIcon } from "@navikt/aksel-icons";
+import cl from "clsx";
+import React, { forwardRef, ChangeEvent } from "react";
+import { BodyShort, ErrorMessage } from "../../typography";
+import { Button } from "../../button";
+
+export interface FileUploadProps {
+  /**
+   * Changes styling when changed.
+   * The "box" variant takes up the full
+   * width of its parent, while the "button"
+   * variant takes up width based on its content.
+   */
+  variant: "box" | "button";
+  /**
+   * Callback called when one or several files
+   * are uploaded either on click or drag-and-drop
+   */
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  /**
+   * ID of the input element. Required to properly
+   * connect input element to potential error message.
+   */
+  inputId: string;
+  /**
+   * Text shown to the user.
+   */
+  label: string;
+  /**
+   * Class name passed to the outermost <div> element.
+   */
+  className?: string;
+  /**
+   * Error message.
+   */
+  error?: string;
+  /**
+   * Indicates if it is possible
+   * to upload multiple files at once.
+   */
+  multiple?: boolean;
+  /**
+   * Indicates which MIME types are
+   * selectable in the file browser.
+   */
+  acceptedMimeTypes?: string[];
+}
+
+/**
+ * A component for uploading files
+ * @see [📝 Documentation](https://aksel.nav.no/komponenter/core/FileUpload)
+ * @see 🏷️ {@link FileUploadProps}
+ * @example
+ * ```jsx
+ * <FileUpload
+ *   onUpload={onUpload}
+ *   variant="button"
+ *   label="Last opp filer her"
+ *   inputId="fileupload-input"
+ * />
+ * ```
+ */
+export const FileUpload = forwardRef<HTMLDivElement, FileUploadProps>(
+  (
+    {
+      onChange,
+      error,
+      inputId,
+      label,
+      className,
+      variant = "box",
+      multiple = true,
+      acceptedMimeTypes
+    },
+    ref
+  ) => {
+    const errorId = `${inputId}-error`
+
+    const accept = acceptedMimeTypes ? acceptedMimeTypes.join(", ") : undefined
+    const ariaDescribedby = error ? errorId : undefined
+    if (variant === "button") {
+      return (
+        <div className={className}>
+          <Button
```

Comment:
Var det endringer for className = 'navds-button...' + Button -> Label ? (eller de var kanskje pending changes 🤔 / spontant i møtet )

Reply:
@JulianNymark  Jeg pusha de endringene nå 😊 Samt en commit hvor jeg skiller på accepted og rejected filer a la slik Polaris gjør det i sin Dropzone.

https://github.com/Shopify/polaris/blob/main/polaris-react/src/components/DropZone/DropZone.tsx

https://polaris.shopify.com/components/selection-and-input/drop-zone

Bygges går rødt, det kan nok ryddes litt i koden, og jeg tror det kan være verdt å skrive noen tester på blant annet dette med accept/reject, men ville bare få pusha det jeg hadde! 😊

Reply:
Tester for accept/reject her https://github.com/navikt/aksel/pull/2504/commits/a794caf6e628d310f920d18ee791c1cc6f91be0b